src/main/java/tatracker/logic/commands/CommandEntry.java

```diff
@@ -0,0 +1,82 @@
+package tatracker.logic.commands;
+
+import java.util.List;
+
+import tatracker.logic.commands.group.AddGroupCommand;
+import tatracker.logic.commands.group.DeleteGroupCommand;
+import tatracker.logic.commands.group.EditGroupCommand;
+import tatracker.logic.commands.module.AddModuleCommand;
+import tatracker.logic.commands.module.DeleteModuleCommand;
+import tatracker.logic.commands.session.AddSessionCommand;
+import tatracker.logic.commands.session.DeleteSessionCommand;
+import tatracker.logic.commands.session.EditSessionCommand;
+import tatracker.logic.commands.student.AddStudentCommand;
+import tatracker.logic.commands.student.DeleteStudentCommand;
+import tatracker.logic.commands.student.EditStudentCommand;
+import tatracker.logic.parser.Prefix;
+import tatracker.logic.parser.PrefixDictionary;
+
+/**
+ * Stores a list of all the commands.
+ */
+public enum CommandEntry {
+    STUDENT_ADD(AddStudentCommand.DETAILS),
+    STUDENT_DELETE(DeleteStudentCommand.DETAILS),
+    STUDENT_EDIT(EditStudentCommand.DETAILS),
+    GROUP_ADD(AddGroupCommand.DETAILS),
+    GROUP_DELETE(DeleteGroupCommand.DETAILS),
+    GROUP_EDIT(EditGroupCommand.DETAILS),
+    MODULE_ADD(AddModuleCommand.DETAILS),
+    MODULE_DELETE(DeleteModuleCommand.DETAILS),
+    SESSION_ADD(AddSessionCommand.DETAILS),
+    SESSION_DELETE(DeleteSessionCommand.DETAILS),
+    SESSION_EDIT(EditSessionCommand.DETAILS),
+    // SORT_STUDENT(SortCommand.DETAILS),
+    // SORT_GROUP(SortGroupCommand.DETAILS),
+    // SORT_MODULE(SortModuleCommand.DETAILS),
```

Comment:
do we have to include filter command ?

Reply:
I disabled this for the meantime. This will be updated with @aakanksha-rai new code